cfme/tests/automate/test_common_methods.py

```diff
@@ -320,3 +321,60 @@ def _finalize():
         assert provision_request.is_succeeded(
             method="ui"
         ), f"Provisioning failed: {provision_request.row.last_message.text}"
+
+
+@pytest.fixture(scope="function")
+def vm_folder(provider):
+    """Create Vm folder on VMWare provider"""
+    folder = provider.mgmt.create_folder(fauxfactory.gen_alpha())
+    yield folder
+
```

Comment:
**REQUIRED:** Remove the folder after test is finished.

Reply:
`folder_` does not seem to be a good start for name, maybe add something like `test_move_vm_folder_{fauxfactory name}` might be longer name but gives more context.

Reply:
@ganeshhubale please review above suggestion, add little more context to name `folder_` .. maybe `test_folder_` .